spice_ev/strategy.py

```diff
@@ -88,12 +88,12 @@ def step(self, event_list=[]):
             # remove event from list
             ev = self.world_state.future_events.pop(0)
 
-            if type(ev) == events.ExternalLoad:
+            if type(ev) == events.FixedLoad:
                 connector = self.world_state.grid_connectors[ev.grid_connector_id]
                 assert ev.name not in self.world_state.charging_stations, (
-                    "External load must not be from charging station")
+                    "Fixed load must not be from charging station")
                 connector.current_loads[ev.name] = ev.value  # not reset after last event
-            elif type(ev) == events.EnergyFeedIn:
+            elif type(ev) == events.LocalEnergyGeneration:
                 assert ev.name not in self.world_state.charging_stations, (
                     "Energy feed-in must not be from charging station")
```

Comment:
```suggestion
                    "Local power generation must not be from charging station")
```

Reply:
Now: "Local energy generation must not be from charging station"